src/stylesheets/elements/_tags.scss

```diff
@@ -359,3 +359,32 @@ Label Title
 .sds-status--disabled {
   @include u-bg("disabled");
 }
+
+//Roadmap tags
+.label-done {
+  background-color: #70e17b;
+  color: #1b1b1b;
+  font-weight: 700;
+}
+
+.label-in-progress {
+  background-color: #ffbe2e;
+  color: #1b1b1b;
+  font-weight: 700;
+}
+
+.label-upcoming {
+  color: #5c5c5c;
+  border-color: #adadad;
+  border-width: 1px;
+  border-style: solid;
+  background-color: transparent;
+}
+
+.label-done,
+.label-in-progress,
+.label-upcoming {
```

Comment:
Could we instead give this generic style a name maybe `roadmap-label` and then on the other labels `@extend .roadmap-label;`